guis/qml/ut/MainUt.qml

```diff
@@ -15,256 +15,290 @@ UITK.Page {
-        desktopLinkDialog.visible = true;
-        root.request = request;
-      } else if(request.message =="refreshContacts"){
-        root.request = request
-        root.requestContentHub = true
-        root.contentType = ContentType.Contacts
-        root.handler = ContentHandler.Source
-        root.selectionType = ContentTransfer.Multiple
-      } else if(request.message =="photo"){
-        root.request = request
-        root.requestContentHub = true
-        root.contentType = ContentType.Pictures
-        root.handler = ContentHandler.Source
-        root.selectionType = ContentTransfer.Single
-      } else if(request.message =="video"){
-        root.request = request
-        root.requestContentHub = true
-        root.contentType = ContentType.Videos
-        root.handler = ContentHandler.Source
-        root.selectionType = ContentTransfer.Single
-      } else if(request.message =="document"){
-        root.request = request
-        root.requestContentHub = true
-        root.contentType = ContentType.Documents
-        root.handler = ContentHandler.Source
-        root.selectionType = ContentTransfer.Single
-      } else if(request.message =="audio"){
-        root.request = request
-        root.requestContentHub = true
-        root.contentType = ContentType.Music
-        root.handler = ContentHandler.Source
-        root.selectionType = ContentTransfer.Single
-      } else if(request.message.includes("[oC]")){
-        root.request = request
-        root.requestContentHub = true
-        root.url = request.message.substring(4)
-        root.contentType = ContentType.Contacts
-        root.handler = ContentHandler.Destination
-        root.selectionType = ContentTransfer.Multiple
-      } else if(request.message.includes("[oP]")){
-        root.request = request
-        root.requestContentHub = true
-        root.url = request.message.substring(4)
-        root.contentType = ContentType.Pictures
-        root.handler = ContentHandler.Destination
-        root.selectionType = ContentTransfer.Single
-      } else if(request.message.includes("[oV]")){
-        root.request = request
-        root.requestContentHub = true
-        root.url = request.message.substring(4)
-        root.contentType = ContentType.Videos
-        root.handler = ContentHandler.Destination
-        root.selectionType = ContentTransfer.Single
-      } else if(request.message.includes("[oD]")){
-        root.request = request
-        root.requestContentHub = true
-        root.url = request.message.substring(4)
-        root.contentType = ContentType.Documents
-        root.handler = ContentHandler.Destination
-        root.selectionType = ContentTransfer.Single
-      } else if(request.message.toLowerCase().includes("call")){
-        root.request = request
-        var callUrl = request.message.substring(4)
-        Qt.openUrlExternally("tel:///"+callUrl);
-        request.dialogAccept();
+    request.accepted = true;
```

Comment:
what do you mean with "method shorthand syntax"?

Reply:
In Javascript this:
```qml
{
  onJavaScriptDialogRequested: function(request) {
    ...
  }
}
```
can be written shorter like this:

```qml
{
  onJavaScriptDialogRequested(request) {
    ...
  }
}
```
Thats called "method shorthand syntax" and I find it more beautiful. :)

Reply:
Not sure though, if it is usable in qml, too.

Reply:
Okey now understand, I think i leave it as is because it's the same in the documentation:
https://doc.qt.io/qt-5/qml-qtwebengine-javascriptdialogrequest.html